terraform/deployments/cdn-analytics/bigquery.tf

```diff
@@ -0,0 +1,128 @@
+resource "google_bigquery_dataset" "fastly_logs" {
+  dataset_id = "fastly_logs"
+
+  location = "europe-west2"
+
```

Comment:
can we also change the billing model to physical
[storage_billing_model](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_dataset#storage_billing_model)

storage_billing_model = "PHYSICAL"